src/wymeditor/editor/image-handler.js

```diff
@@ -575,6 +579,7 @@ WYMeditor.ImageHandler.prototype._onImgMousedown = function (evt) {
     }
 
     // returning false here prevents drag of image
+    // except for in IE8
```

Comment:
not really an IE8 person at all and just throwing random ideas here.  does doing anything in dragstart handler help?